index.css

```diff
@@ -1,53 +1,63 @@
+*{
+    margin: 0;
+    padding: 0;
+}
```

Comment:
This is good practice also consider adding box-sizing: border-box; You can learn more on the same on W3Schools